### DISCUSSION TITLE
Collab Request

### DISCUSSION THREAD

**Discussion by MAYKELL07** [BochilGaming/games-wabot#130] [Ideas]
**Hallo bochil, gw bisa ngefix berberapa fitur mu. aku juga ingin membantu agar bot nya tambah MANTAB**

**Comment by BochilGaming** [comment]
> **Hallo bochil, gw bisa ngefix berberapa fitur mu. aku juga ingin membantu agar bot nya tambah MANTAB**

Hebat @MAYKELL07 mama mu pasti bangga👍, jika mau menambahkan atau mengefix fitur bisa melalui pull request😁

**Reply by MAYKELL07** [reply]
ada nomor wa? ntik gw chat

**Reply by BochilGaming** [reply]
> ada nomor wa? ntik gw chat

tanya in aja di [gc](https://github.com/BochilGaming/games-wabot/issues/128) nya :v

**Reply by unx21** [reply]
> > **Hallo bochil, gw bisa ngefix berberapa fitur mu. aku juga ingin membantu agar bot nya tambah MANTAB**
> 
> Hebat @MAYKELL07 mama mu pasti bangga👍, jika mau menambahkan atau mengefix fitur bisa melalui pull request😁

Bangga banget mamah nya, saya yang baca juga bangga fitur games-wabot bakal di figs

**Reply by botstylee** [reply]
Wah gw ketinggalan berapa abad nih lihat ABG jago pasti ortunya bangga ya Hua :v

**Comment by Thecaracter** [comment]
![Uploading Screenshot (619).png…]()
bang kok gini ya

**Reply by Thecaracter** [reply]
gua kebingungann soalnya bang
